Classes/SchedulerTasks/MailNotifier.php

```diff
@@ -182,18 +175,20 @@ private function sendRemindersToOrganizers(\Tx_Seminars_OldModel_Event $event, s
 
         /** @var \Tx_Seminars_OldModel_Organizer $organizer */
         foreach ($event->getOrganizerBag() as $organizer) {
-            /** @var \Tx_Oelib_Mail $eMail */
-            $eMail = GeneralUtility::makeInstance(\Tx_Oelib_Mail::class);
-            $eMail->setSender($sender);
-            $eMail->setReplyTo($replyTo);
-            $eMail->setSubject($subject);
-            $eMail->addRecipient($organizer);
-            $eMail->setMessage($this->customizeMessage($messageKey, $event, $organizer->getName()));
+            /** @var MailMessage $mail */
+            $mail = GeneralUtility::makeInstance(MailMessage::class);
+            $mail->setFrom($sender);
+            $mail->setTo($organizer);
```

Comment:
You'll also need to extract email address and name from sender and organizer to pass them into the new email message.